src/compile/model.ts

```diff
@@ -209,7 +209,7 @@ export abstract class Model {
 
     this.description = spec.description;
     this.transforms = normalizeTransform(spec.transform ?? []);
-    this.layout = isUnitSpec(spec) || isLayerSpec(spec) ? {} : extractCompositionLayout(spec, type, config);
+    this.layout = type === 'layer' || type === 'unit' ? {} : extractCompositionLayout(spec, type, config);
```

Comment:
why is this related?

Reply:
extractCompositionLayout now has a type for column while it was any before.

Reply:
still irrelevant to this PR, but sure I can ignore it.

Reply:
Okay